R/plot.R

```diff
@@ -706,27 +702,18 @@ plot_quantile_coverage <- function(scores,
 #' @export
 #' @examples
 #' library(ggplot2)
-#' df <- data.frame(
-#'   model = rep(c("model1", "model2", "model3"), each = 10),
-#'   id = rep(1:10),
-#'   interval_score = abs(rnorm(30, mean = rep(c(1, 1.3, 2), each = 10))),
-#'   ae_median = (abs(rnorm(30)))
-#' )
-#'
 #' scores <- score(example_quantile)
 #' pairwise <- pairwise_comparison(scores, by = "target_type")
-#' plot_pairwise_comparison(pairwise) +
+#' plot_pairwise_comparison(pairwise, type = "mean_scores_ratio") +
 #'   facet_wrap(~target_type)
 
 plot_pairwise_comparison <- function(comparison_result,
-                                     type = c("mean_scores_ratio", "pval", "together"),
-                                     smaller_is_good = TRUE) {
+                                     type = c("mean_scores_ratio", "pval")) {
```

Comment:
`match.arg` to check allowed options used.

Reply:
It does use match.arg in line 782 - is that different from what you meant?

Reply:
For type? If it does then no that is exactly what I meant